speech/cloud-client/transcribe.py

```diff
@@ -57,13 +55,48 @@ def main(speech_file):
     # [END send_request]
 
 
+def transcribe_gcs(gcs_uri):
+    """Transcribes the audio file specified by the gcs_uri."""
+    # [START authenticating_gcs]
```

Comment:
+1, these aren't even referenced anywhere in the docs yet, I'd just stuck to the inherited convention.